.docker/Dockerfile.alpine

```diff
@@ -88,7 +88,7 @@ ENV NODE_RED_VERSION=$NODE_RED_VERSION \
 # ENV NODE_RED_ENABLE_SAFE_MODE=true    # Uncomment to enable safe start mode (flows not running)
 # ENV NODE_RED_ENABLE_PROJECTS=true     # Uncomment to enable projects option
 
-# User configuration directory volume
+# User configuration directory volume (if required)
```

Comment:
would recommend just to remove it instead of putting it into comment

Reply:
ok